layers/+completion/helm/packages.el

```diff
@@ -319,8 +319,9 @@
       (defalias
         'spacemacs/helm-project-do-grep-region-or-symbol
         'helm-projectile-grep))
-    :config (define-key helm-projectile-find-file-map
-              (kbd "RET") 'spacemacs/helm-find-files-windows)))
+    :config (when (configuration-layer/package-used-p 'winum)
```

Comment:
I think it should be wrapped around (with-eval-after-load 'winum ... ) and not (when ... ).

Reply:
I'm just following what @syl20bnr did in e5d8603 . The original issue was that this layer can be loaded without winum (like with `(helm :packages (not winum))`), so the approach in this change seems most appropriate.